Dockerfile.worker

```diff
@@ -3,4 +3,4 @@ ARG DOCKER_IMAGE=nimblehq/ruby_google_scraper
 ARG DOCKER_REGISTRY=docker.io
 FROM ${DOCKER_REGISTRY}/${DOCKER_IMAGE}:${BRANCH_TAG}
 
-CMD ./bin/sidekiq.sh
+CMD bundle exec sidekiq -C config/sidekiq.yml
```

Comment:
how about create a `bin/worker.sh` file

and inside that file, it includes the `bundle exec sidekiq -C config/sidekiq.yml` @gutakk 🤔

Reply:
It's a small thing so you can decide to change it or not ;) The PR LGTM